app/src/main/java/com/google/maps/android/compose/RecompositionActivity.kt

```diff
@@ -65,8 +64,7 @@ class RecompositionActivity : ComponentActivity() {
         cameraPositionState: CameraPositionState = rememberCameraPositionState(),
         content: @Composable () -> Unit = {},
     ) {
-        var location by remember { mutableStateOf(singapore) }
-        val singaporeState = MarkerState(position = location)
+        val markerState = remember { MarkerState(position = singapore) }
```

Comment:
Same change to `rememberMarkerState()` here, to retain position across configuration changes.
```suggestion
        val markerState = rememberMarkerState(position = singapore)
```